### PR TITLE
 Add Ficolo PRBUILDER  configuration 

### PR DESCRIPTION
**Pull request by karim20230**
Configure NixOS for "prbuilder" with Docker support, user management, and network settings.

### CODE REVIEW

**Review by cazfi — APPROVED** [review]

File: hosts/prbuilder/configuration.nix [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+      user-cazfi
+      user-karim
+    ])
+    ./disk-config.nix
+  ];
+  nixpkgs.hostPlatform = lib.mkDefault "x86_64-linux";
+   # List packages installed in system profile
+  environment.systemPackages = with pkgs; [
+    git
+    docker
```
Comment (joinemm) [inline hosts/prbuilder/configuration.nix:30]: docker is already installed by the `virtualisation.docker.enable` option so this is not needed.

File: hosts/prbuilder/configuration.nix [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+  ];
+# docker daemon running
+virtualisation.docker.enable=true;
```
Comment (joinemm) [inline hosts/prbuilder/configuration.nix:33]: should be properly indented

**Review by joinemm — CHANGES_REQUESTED** [review]

File: tasks.py [hunk 3]
```diff
@@ -85,6 +85,7 @@ class TargetHost:
         "build3-ficolo": TargetHost(
             hostname="172.18.20.104", nixosconfig="ficolobuild"
         ),
+        "prbuilder": TargetHost(hostname="172.18.20.106", nixosconfig="prbuilder"),
```
Comment (mkaapu) [inline tasks.py:88]: minor, but maybe keep to the same style us above.

**Comment by karim20230** [comment]
addressed cmt and installed  emacs to make @cazfi  life easier!

**Review by joinemm — APPROVED** [review]

**Pull request merged**